lightwood/mixer/regression.py

```diff
@@ -41,10 +63,24 @@ def fit(self, train_data: EncodedDs, dev_data: EncodedDs) -> None:
         log.info(f'Regression based correlation of: {self.model.score(X, Y)}')
 
     def partial_fit(self, train_data: EncodedDs, dev_data: EncodedDs) -> None:
+        """
+        Fits the Neural mixer on some data, this refits the model entirely rather than updating it
+
+        :param train_data: The EncodedDs on which to train the network
+        :param dev_data: Data used for early stopping and hyperparameter determination
+        """
         self.fit(train_data, dev_data)
 
     def __call__(self, ds: EncodedDs,
                  args: PredictionArguments = PredictionArguments()) -> pd.DataFrame:
+        """
+        Make predictions based on datasource similar to the one used to fit (sans the target column)
```

Comment:
This doesn't even make sense, what do you mean similar? The same data structure? This should be obvious.